TombEngine/Objects/Generic/generic_objects.cpp

```diff
@@ -97,6 +98,12 @@ static void StartObject(ObjectInfo* object)
 		}
 	}
 
+	for (int objectNumber = ID_PUSHABLE_OBJECT1; objectNumber <= ID_PUSHABLE_OBJECT10; objectNumber++)
+		InitPushableObject(object, objectNumber);
+
```

Comment:
Why does this part of your code not follow suit with other objects? For other generic objects you can set things like the hit effect or shadow type but not with this?

Reply:
I also saw a bit weird, if you follow those initpushableobject functions, you will eventually reach to another initialization block that similar to the other objects. I don't know why it was separated in that way, I didn't notice anything "so" special, but well, it didn't hurt so I kept it.

(It looks "added" because I moved it from TR5 objects into the general objects).